features/official_copy_legal_text.feature

```diff
@@ -0,0 +1,41 @@
+@official_copy
+Feature: Legal requirement updates made to Official Copy of the Register
```

Comment:
Scenarios in functional tests should represent use cases, not small aspects of one use case. I could agree on having them separate for now, but some of them have to be rephrased, so that they talk about the expected behaviour instead of what changes have been made.